ngraph/frontend/onnx_import/src/op/resize.cpp

```diff
@@ -28,6 +29,124 @@ namespace ngraph
         {
             namespace
             {
+                static const std::unordered_set<std::string> supported_modes = {
+                    "nearest", "linear", "cubic"};
+
+                static const std::unordered_set<std::string> supported_transforms = {
+                    "half_pixel",
+                    "pytorch_half_pixel",
+                    "align_corners",
+                    "asymmetric",
+                    "tf_half_pixel_for_nn"};
+
+                using InterpolateMode = ngraph::op::v4::Interpolate::InterpolateMode;
+                static const std::map<std::string, InterpolateMode> interp_mode_map = {
+                    {"nearest", InterpolateMode::nearest},
+                    {"linear", InterpolateMode::linear_onnx},
+                    {"cubic", InterpolateMode::cubic}};
+
+                using Transform_mode = ngraph::op::v4::Interpolate::CoordinateTransformMode;
+                static const std::map<std::string, Transform_mode> transform_mode_map = {
+                    {"half_pixel", Transform_mode::half_pixel},
+                    {"pytorch_half_pixel", Transform_mode::pytorch_half_pixel},
+                    {"align_corners", Transform_mode::align_corners},
+                    {"asymmetric", Transform_mode::asymmetric},
+                    {"tf_half_pixel_for_nn", Transform_mode::tf_half_pixel_for_nn}};
+
+                using Nearest_mode = ngraph::op::v4::Interpolate::NearestMode;
+                static const std::map<std::string, Nearest_mode> nearest_mode_map = {
+                    {"round_prefer_floor", Nearest_mode::round_prefer_floor},
+                    {"round_prefer_ceil", Nearest_mode::round_prefer_ceil},
+                    {"floor", Nearest_mode::floor},
+                    {"ceil", Nearest_mode::ceil},
+                    {"simple", Nearest_mode::simple}};
```

Comment:
There's no `simple` mode in the docs https://github.com/onnx/onnx/blob/master/docs/Operators.md#Resize

Reply:
But this mode there is in ONNX sources (see lines 228 and 235-244 of https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/providers/cpu/tensor/upsample.h). Accordingly  comments at lines 237-238 of the mentioned file, this nearest_mode is used for opset versions older than 11.

Reply:
This is Microsoft's implementation of the ONNX standard. Officially there's no such mode in the standard itself in none of the opsets. I'm not sure if we should be doing the same thing. @postrational any thoughts?

Reply:
Deleted.